pkg/infrabin/config.go

```diff
@@ -38,6 +39,8 @@ func ReadConfiguration() {
 	// Config file should be config.yaml
 	viper.SetConfigName(DefaultConfigName)
 	viper.SetConfigType(DefaultConfigType)
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+	viper.AutomaticEnv()
```

Comment:
Can you add a note about why this change?

Reply:
left a note in the PR description